src/server/auth/cmds/cmds.go

```diff
@@ -490,6 +490,32 @@ func UseAuthTokenCmd() *cobra.Command {
 	return cmdutil.CreateAlias(useAuthToken, "auth use-auth-token")
 }
 
+// GetOneTimePasswordCmd returns a cobra command that lets a user get an OTP.
+func GetOneTimePasswordCmd() *cobra.Command {
+	getOneTimePassword := &cobra.Command{
+		Use:   "{{alias}} <username>",
+		Short: "Get a one-time password that authenticates the holder as \"username\"",
+		Long:  "Get a one-time password that authenticates the holder as \"username\"",
+		Run: cmdutil.RunFixedArgs(1, func(args []string) error {
+			subject := args[0]
+			c, err := client.NewOnUserMachine("user")
+			if err != nil {
+				return fmt.Errorf("could not connect: %v", err)
+			}
+			defer c.Close()
+			resp, err := c.GetOneTimePassword(c.Ctx(), &auth.GetOneTimePasswordRequest{
+				Subject: subject,
+			})
+			if err != nil {
+				return grpcutil.ScrubGRPC(err)
+			}
+			fmt.Println(resp.Code)
+			return nil
+		}),
+	}
+	return cmdutil.CreateAlias(getOneTimePassword, "auth get-one-time-password")
```

Comment:
how would you feel about `get-otp`? Just because `get-one-time-password` seems kind of long?

Reply:
Yea, makes sense. I think my original logic for keeping it long was that the long form is how it is defined in the auth proto service and wanted to keep it consistent, but that reasoning feels kinda lame on second thought. Fixed.